diff --git a/dist/chrome-launcher.js b/dist/chrome-launcher.js
index d7fb4cf753f817e241554438b317ec333af2e7fc..7fc7bb6f10a2001922131ce9d90a500476294f7b 100644
--- a/dist/chrome-launcher.js
+++ b/dist/chrome-launcher.js
@@ -363,8 +363,13 @@ class Launcher {
         }
         // backwards support for node v12 + v14.14+
         // https://nodejs.org/api/deprecations.html#DEP0147
-        const rmSync = this.fs.rmSync || this.fs.rmdirSync;
-        rmSync(this.userDataDir, { recursive: true, force: true, maxRetries: 10 });
+        try {
+            const rmSync = this.fs.rmSync || this.fs.rmdirSync;
+            rmSync(this.userDataDir, { recursive: true, force: true, maxRetries: 10 });
+        }
+        catch (err) {
+            log.warn('ChromeLauncher', `Failed to delete tmp dir ${this.userDataDir}: ${err.message}`);
+        }
     }
 }
 ;
